### PR TITLE
Feature/angular 9

### PR DESCRIPTION
**Pull request by mario-subo**
Upgrades the entire project to angular 9

passed scripts:
- [x] npm run docs
- [x] npm run test 
- [x] npm run build-lib
- [x] npm run build-docs

### CODE REVIEW

File: projects/ng-components-docs/tsconfig.app.json [hunk 1]
```diff
@@ -11,5 +11,6 @@
             "path": ["lib/src/button/*"]
         }
     },
-    "exclude": ["test.ts", "**/*.spec.ts"]
+    "files": ["src/main.ts", "src/polyfills.ts"],
```
Comment (omarVodiaK) [inline projects/ng-components-docs/tsconfig.app.json:14]: is it ok to remove exclude ?

Comment (mario-subo) [reply projects/ng-components-docs/tsconfig.app.json:14]: this was done automatically by ng update script. looks like they are moving away from exclude to include or files instead. But to be honest I'm not sure. I need to test out all the build scripts etc

**Pull request merged**